### ISSUE TITLE
Chroma-hnswlib-0.7.3 issue with latest update

### ISSUE THREAD

**Issue by jajaborsthan** [sm18lr88/Open-Interpreter-Termux#2]
Using android13 on a custom rom. Haven't had any issues until the latest update. Don't think it's strictly an OI issue. 

Error message attached.
[chroma-hw.pdf](https://github.com/sm18lr88/Open-Interpreter-Termux/files/13038707/chroma-hw.pdf)

**Comment by sm18lr88** [comment]
Yup. Just tried a fresh install with my one-liner and got the same error. I'll work on it within a few days and update you when I fix it. Thanks for letting me know.
